Packages/com.unity.inputsystem/InputSystem/Plugins/Android/InputSettingsAndroid.cs

```diff
@@ -0,0 +1,51 @@
+#if UNITY_EDITOR || UNITY_ANDROID || PACKAGE_DOCS_GENERATION
+using System;
+
+namespace UnityEngine.InputSystem
+{
+    public partial class InputSettings
+    {
+        /// <summary>
+        /// Project-wide input settings for the Android platform.
+        /// </summary>
+        [Serializable]
+        public class AndroidSettings
+        {
+            /// <summary>
+            /// Determines whether pressing the back button on the device should leave the app or not.
+            /// </summary>
+            /// <remarks>
+            /// On Android 13 and above, you can use this option in combination with `PlayerSettings.Android.predictiveBackSupport`
+            /// to enable predictive back animations.
+            /// </remarks>
+            public bool backButtonLeavesApp
```

Comment:
In UnityEngine.Input there is guidance about manually responding to back button, is something similar needed here maybe?

Reply:
5a58763

Reply:
d4f551d